examples/compose/docker-compose.yml

```diff
@@ -124,7 +124,7 @@ services:
     - -c
     - ' /vt/bin/vtctld -topo_implementation consul -topo_global_server_address consul1:8500
       -topo_global_root vitess/global -cell test -workflow_manager_init -workflow_manager_use_election
-      -service_map ''grpc-vtctl'' -backup_storage_implementation file -file_backup_storage_root
+      -service_map ''grpc-vtctl,grpc-vtctld'' -backup_storage_implementation file -file_backup_storage_root
```

Comment:
I ran into the below error when running `vtctldclient.`

```
vitess@88dc42cad55b:/$ ./vt/bin/vtctldclient --server localhost:15999 GetBackups test_keyspace/-80
ERROR: logging before flag.Parse: E0622 08:03:08.797336      69 syslogger.go:149] can't connect to syslog
E0622 08:03:08.846768      69 main.go:42] rpc error: code = Unimplemented desc = unknown service vtctlservice.Vtctld
```

I guess you need to add `grpc-vtctld` to vtctld's service_map.

- ref. https://github.com/vitessio/vitess/issues/7894

Reply:
Yes, that is correct